properties/openelis/hibernate.properties

```diff
@@ -0,0 +1,3 @@
+hibernate.connection.url=jdbc:postgresql://${OPENELIS_DB_HOST}:5432/clinlims
```

Comment:
Database name could be parametrized right?

Reply:
Database name is hard-coded in different places in OpenELIS

Reply:
ok